File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long  p=0;
        int s =x;
        if(x < 0) return false;
        while(x != 0){
            int r = x%10;
            p = p*10+r;
            x = x/10;
        }
        if((int)p == s){
            return true;
        }
        return false;

    }
};

File: 9-palindrome-number/palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        if(x<0) return false;
        int r , n = 0;
        int s = x;
        while(x>0){
            r = x %10;
            n = n*10 + r;
            x = x/10;
        }
        if(s == n){
            return true;
        }
        return false;
    }
}